Example/Controllers/UserSettingsTableViewController.swift

```diff
@@ -30,6 +31,16 @@ class UserSettingsTableViewController: UITableViewController, UIImagePickerContr
         self.userProfileImage = profileImage
     }
 
+    func addTapGestureForImage() {
+        let tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(imageTapped(tapGestureRecognizer:)))
+        imageView.isUserInteractionEnabled = true
+        imageView.addGestureRecognizer(tapGestureRecognizer)
```

Comment:
why not using a button? Less code is better to me ^^

Reply:
Lol. 👍 . Added in refactor To-Do's.